docker-compose.yml

```diff
@@ -140,7 +140,7 @@ services:
       dockerfile: docker/dev.Dockerfile
     environment:
       - CELERY_BROKER_URL=amqp://guest:guest@rabbitmq:5672//
-      - DATABASE_URL=mysql://root@mysql:3306/treeherder
+      - DATABASE_URL=mysql://root@mysql:localhost:3306/treeherder
```

Comment:
No, that's not correct.

The host inside of the Docker network is `mysql`.

Abort the Pulse ingestion if os.environ["DATABASE_URL"] is different than the default value.

See that SKIP_INGESTION was added to manually preventing the manual ingestion from working.

How can you test that this works?
`export DATABASE_URL=<some_non_default_value>`
Run `docker-compose up`
Pulse ingestion outputs a message that it won't be ingesting data.

Reply:
Thank you, it should work now!

Reply:
Gives me (KeyError: 'DATABASE_URL') on [travis](https://travis-ci.org/github/mozilla/treeherder/jobs/678252483#L684)